config/wdio/wdio.conf.js

```diff
@@ -11,6 +11,7 @@ const ip = process.env.WDIO_EXTERNAL_HOST || localIP.address();
 const externalPort = process.env.WDIO_EXTERNAL_PORT || 8080;
 const internalPort = process.env.WDIO_INTERNAL_PORT || 8080;
 const ci = process.env.TRAVIS || process.env.CI;
+const bail = process.env.WDIO_BAIL || ci;
```

Comment:
Are we enforcing that ci builds always bail? 

Currently, if a user provided WDIO_BAIL=false this would always evaluate to the value of ci.

Reply:
That's correct. CI builds at present have chosen to have bail set to 1.